ajax/libs/layui/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "layui",
+  "version": "2.2.3",
+  "description": "经典模块化前端框架",
+  "filename": "layui.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/sentsin/layui.git"
+  },
+  "author": "贤心",
+  "homepage": "http://www.layui.com",
+  "keywords": [
+    "layui",
+    "ui"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/sentsin/layui.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
```

Comment:
In versions below 2.0.0 there are no `dist `folder.There are a `build `folder, thus please another pattern here to fetch also `build `directory